requirements.txt

```diff
@@ -1,43 +1,2 @@
-requests~=2.25.1
-requests-oauthlib==1.3.0
-requests-toolbelt==0.9.1
-numpy==1.22.0
-pytest~=6.2.5
-biopython==1.78
-urllib3~=1.26.5
-py~=1.11.0
-colorama~=0.4.4
-pillow~=8.3.2
-scipy~=1.6.1
-pip~=22.0.4
-attrs~=20.3.0
-toml~=0.10.2
-wheel~=0.37.1
-jinja2~=3.1.2
-pytz~=2021.1
-cffi~=1.14.5
-cython~=0.29.22
-setuptools~=62.1.0
-nose~=1.3.7
-wcwidth~=0.2.5
-pluggy~=1.0.0
-deprecated~=1.2.12
-iniconfig~=1.1.1
-pexpect~=4.8.0
-certifi~=2020.12.5
-chardet~=4.0.0
-oauthlib~=3.1.0
-pyparsing~=2.4.2
-packaging~=20.9
-zipp~=3.8.0
-future~=0.18.2
-pandas~=1.2.3
-ipython~=7.21.0
-cryptography~=38.0.1
-matplotlib~=3.3.4
-pygments~=2.8.1
-markupsafe~=2.1.1
-ptyprocess~=0.7.0
-six~=1.15.0
-pycparser~=2.20
-wrapt~=1.12.1
\ No newline at end of file
+pipebio==1.0.0
+pytest~=6.2.5
```

Comment:
Very nice. Is it worth deleting the library / models files since they're not used anymore?

Reply:
They are deleted, aren't they?

Reply:
Maybe I'm mistaken sorry, I probably interpreted the diff wrong